### PR TITLE
monorepo: add dependabot support

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Adds a dependabot.yml file from https://github.com/celestiaorg/optimism/blob/celestia-develop/.github/dependabot.yml

Based on conversation in https://github.com/ethereum-optimism/optimism/issues/5764

Co-authored-by: mjsevey@gmail.com

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  # Check for updates on our docker image and for celestia-node
```
Comment (refcell) [inline .github/dependabot.yml]: We should remove this right?
```suggestion
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    labels:
+      - dependencies
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
+    labels:
+      - dependencies
+  # Check for updates on celestia packages
```
Comment (refcell) [inline .github/dependabot.yml]: ibid
```suggestion
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  # Check for updates on our docker image and for celestia-node
+  - package-ecosystem: docker
+    directory: "/ops-bedrock"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
+    labels:
+      - dependencies
```
Comment (refcell) [inline .github/dependabot.yml]: nit
```suggestion
      - dependabot
```

File: .github/dependabot.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
+    labels:
+      - dependencies
```
Comment (refcell) [inline .github/dependabot.yml]: nit
```suggestion
      - dependabot
```

File: .github/dependabot.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  # Check for updates on celestia packages
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
+    labels:
+      - dependencies
```
Comment (refcell) [inline .github/dependabot.yml]: nit
```suggestion
      - dependabot
```

File: .github/dependabot.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  # Check for updates on celestia packages
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
```
Comment (refcell) [inline .github/dependabot.yml:73]: ibid
```suggestion
    open-pull-requests-limit: 10
    reviewers
      - "ethereum-optimism/infra-reviewers"
```

File: .github/dependabot.yml [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    labels:
+      - dependencies
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
```
Comment (refcell) [inline .github/dependabot.yml:15]: ibid
```suggestion
    open-pull-requests-limit: 10
    reviewers
      - "ethereum-optimism/infra-reviewers"
```

File: .github/dependabot.yml [hunk 8]
```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  # Check for updates on our docker image and for celestia-node
+  - package-ecosystem: docker
+    directory: "/ops-bedrock"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
```
Comment (refcell) [inline .github/dependabot.yml:7]: Should we add infra reviewers here?
```suggestion
    open-pull-requests-limit: 10
    reviewers
      - "ethereum-optimism/infra-reviewers"
```

**Review by mslipper — APPROVED** [review]

**Pull request merged**